workspaces/ui-v2/src/lib/diff-description-interpreter.ts

```diff
@@ -87,6 +92,7 @@ export async function descriptionForShapeDiff(
 
   const getJsonBodyToPreview = (interaction: any) => {
     const body =
+      (location.inQuery && interaction.query.body) ||
```

Comment:
I'm not sure if this is correct, or how this will line up. This will also need to be checked when we integrate the two parts

Reply:
I'm not sure either, the integration will tell us for sure.